lib/miq_automation_engine/service_models/miq_ae_service_orchestration_stack.rb

```diff
@@ -30,5 +31,9 @@ def normalized_live_status
     rescue MiqException::MiqOrchestrationStackNotExistError => err
       ['not_exist', err.message]
     end
+
+    def self.refresh(manager_id, manager_ref)
```

Comment:
@Ladas Code looks fine, but one last question: What is the use case for having the class method for refresh from automate?

Reply:
Right, so the new targeted refresh is not queuing MIQ models. Because that requires that the OrchestrationStack record was already refreshed. 

So instead we have a generic ManagerRefresh::Target (https://github.com/Ladas/manageiq/blob/300deb4c9aa3f7d35401f4a61fee980921142490/app/models/orchestration_stack.rb#L163), that we can queue for the refresh. And we call it as a class method, since this doesn't have to be existing OrchestrationStack record in MIQ DB.